docs/inventory/inventory.md

```diff
@@ -39,27 +39,34 @@ With the Nornir Nautobot Inventory plugin you have the option of using all of th
 
 ## Filtering Examples
 
-For each of the examples the sites correspond to the first three letters of the devices below:
+For each of the examples the locations correspond to the first three letters of the devices below:
 
 ```python
 >>> nautobot.dcim.devices.all()
 [den-rtr01, den-rtr02, grb-rtr01, msp-rtr01, msp-rtr02, nyc-rtr01, nyc-rtr02]
 ```
 
-### Filtering Example: Select from single site
+### Filtering Example: Select from single location
+
+To filter from a single location you can use the filter location to get the devices at a single location based on the **Primary Key** for the location:
+
+> NOTE: Location names do not guarantee uniqueness in Nautobot 2.0.
+> 
+> See this document for more information:
+> 
+> https://docs.nautobot.com/projects/core/en/next/development/apps/api/platform-features/uniquely-identify-objects/
 
-To filter from a single site you can use the filter site to get the devices at a single site based on the **slug** for the site:
 
 ```python
-site = "msp"
+location = "db913e3b-cbe0-4463-addc-816ba6a20100"
```

Comment:
I thought I noted this already, but it would appear not.

This is not a useable experience imho. This would not work between environments. I understand the "why", on a technical level, in that we cannot guarantee uniqueness, but it I do not believe this is usable.

Reply:
What do you suggest then? Stick to the name? Query still can be done with `name`, but uniqueness can't be guaranteed. We can add a note, that if uniqueness is a concern then PK (id) or combination of attributes which guarantee queering unique object should be used.

Reply:
Yea, stick to the name and the note.

Reply:
Requested changes are in #116